### ISSUE TITLE
Guidelines for rewards / experience

### ISSUE THREAD

**Issue by djigneo** [our-stories/west-marches#25]
Should there be a semi-formalized structure for expectations of rewards for players/characters as a result of playing a session?

In my opinion, there are many different forms of reward for characters and players:
- Wealth (coin, weapons, equipment, magical items)
- Diplomatic gain / new ally
- Magical power
- Character growth / development
- Experience points

Some organized play groups have a type of currency so that players have a certain amount of latitude in selecting their rewards for participating in a session with mechanisms such as Treasure points (Adventurer's League).

**Comment by zmoazeni** [comment]
Great questions! I was thinking of keeping it simple, but I can already spot a few drawbacks. _Note: I think anything we decide will have drawbacks._

**I was going to go with straight up DMG XP per character:** based on the encounters/challenges they overcome. XP will only not just be awarded for fighting. As I mentioned in the character issue, if a player gives their character a serious flaw that they have to struggle with under pressure they should be rewarded with an XP bonus _(e.g. Glamdrig is terrified of spiders. Anytime he notices a spider around him, he immediately takes a disadvantage on all rolls)._

**All XP was will be awarded after the session ends.** I'm not going to tally up everything in the middle of the game.

**Equipment cannot be shared among your own characters.** I don't think it makes sense for us to lock down Player-Player trading of equipment. That could even be a fun opportunity for text RP here on GitHub with an entire marketplace. And ultimately I don't want to moderate it too much. But I like the idea that if I earned something with my character Roderick, I need to use it, sell it, or give it to some other Player that would find it worthwhile.

All of that said, I do want status/renown to be a thing. **I want character's actions to affect Waldenstone and the surrounding areas.** Same with earning patrons, and also enemies.

Thing is, I just don't know how to formalize it, or even if we should. The more rules that are in place, the more of a pain in the butt this whole thing becomes in order to manage. But I am curious about thoughts.

The one big gap in this, which is the differences in GMs. **One GM may award more magical items, gold, etc than another**. I don't know how to maintain a consistency there other than to be open about expectations and still leave it to their discretion. I personally still want magical items to _mean_ something. But at the same time, I have the expectation a single adventure should yield at least 2 or 3 items for the entire party (not individually). So perhaps listing out rules of thumb would be better there. To at least to start with.

**Comment by djigneo** [comment]
I suggest having XP tallied in one consistent place / way, so a GM can award XP at the end of a session and it leaves a history instead of being on players to keep it tallied.

I'll let others speak on this topic, but I think a rule of thumb is a good idea to post so to give the GMs a starting point.

**Comment by zmoazeni** [comment]
@djigneo What if the GMs post the XP (as well as any valuables and magic items awarded) to the session issue after the session ends?

**Comment by crimsonknave** [comment]
If we are going to have multiple GMs, I think we need some sort of way to track reputation/renown (both positive and negative). Otherwise it'll be hard for everyone to really keep in mind that Character X has saved the town a bunch and Character Y hasn't really done anything helpful.

**Comment by djigneo** [comment]
> @djigneo What if the GMs post the XP (as well as any valuables and magic items awarded) to the session issue after the session ends?

Yes, and some way of having it annotated on a character so that it's consolidated as well.



> If we are going to have multiple GMs, I think we need some sort of way to track reputation/renown (both positive and negative). Otherwise it'll be hard for everyone to really keep in mind that Character X has saved the town a bunch and Character Y hasn't really done anything helpful.

I think that's an interesting idea, but it could be susceptible to "GM fairness" much in the same way as what I was mentioning in the OP, unless there's a baseline.

**Comment by crimsonknave** [comment]
Hmm, that's reasonable. I think we could have everyone involved decide if the adventure earned any reputation/renown from the town and then everyone involved would have it. If we're still having a central party page with names, links to sheets we could add in reputation/renown there and it would just get updated after each adventure.

**Comment by zmoazeni** [comment]
I think even just tracking Renown in their character sheets and the github wiki page for that character may be a good thing. With a loose rubric.

- Complete and adventure to help Waldenstone? Everyone earns one renown.
- Complete a very difficult adventure. 2 renown.
- Steal or cheat a local. -1 renown 
- Murder a local in cold blood. -2 renown.

I bet we could put together a rough guide on how to use it and commit that too.

- Most shop keeps require 0 renown.
- 5+ and most shops give you a 10% discount
- < 0 renown and most shops will refuse you or charge you an extra 10%
- Thieves guilds, fences, and other shadowy organizations may require <= -5 renown
- 20 renown (positive or negative) starts attracting the attention of powerful people on either side.

Renown can also be figured out if the Player(s) want to buy or build a keep. Within the town border or outside. I don't know numbers yet, but we could put something together that characters could strive for.

What do you all think of the idea. And the numbers I just threw together?

**Comment by crimsonknave** [comment]
I actually realized earlier today that the DMG has optional rules for renown in organizations: https://www.dndbeyond.com/sources/dmg/a-world-of-your-own#Renown

I think that mostly makes sense, but I do feel that there should be more nuance to renown than good/bad. A character could easily save the town a bunch but also be really shady and in the thieves guild's good graces.
